easybuild/easyconfigs/g/Ghostscript/Ghostscript-9.23-GCCcore-7.3.0-cairo-1.14.12.eb

```diff
@@ -0,0 +1,48 @@
+easyblock = 'ConfigureMake'
+
+name = 'Ghostscript'
+version = '9.23'
+cairover = '1.14.12'
+versionsuffix = '-cairo-%s' % cairover
```

Comment:
@crubb You can drop the `-cairo-*` versionsuffix, this was only done to discriminate between `Ghostscript-9.22-GCCcore-6.4.0.eb` which uses `cairo` `1.14.10` when the need arose to have a `Ghostscript` with a newer version of `cairo`.

So, just drop these two lines & rename the easyconfig.